app/preparing_for_the_great_battle.py

```diff
@@ -0,0 +1,25 @@
+from app.structuring_the_knight_personality import Knight
+
+
+class BeforeBattle:
```

Comment:
The class name `BeforeBattle` does not make much sense; in case you use it only as a function collection or a collection of static methods, think of writing separate functions and collecting them in separate `support_functons.py` file.

Reply:
👍